Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Development Docker file
+FROM node:18-alpine as development
+
+WORKDIR /async
+
+# Install development dependencies
+COPY package.json package-lock.json ./
+RUN npm install
+
+# Copy the rest of the application files
+COPY . .
```

Comment:
I was wondering what is it for, 

you also have `FROM node:18-alpine as development`

are these some leftovers from initial multi stage image that you worked on?